SSNFSd/ssnfsworker.cpp

```diff
@@ -171,6 +173,22 @@ QString SSNFSWorker::getPerms(QString path, qint32 uid) {
     }
 }
 
+ QByteArray SSNFSWorker::isValidPath(QByteArray path){
+    QString newPath = QTextCodec::codecForMib(1015)->toUnicode(path);
+    QStringList directories = newPath.split("/");
+    QStack<QString> normalizer;
+    // Assuming the path starts and ends with /
```

Comment:
Don't assume anything. In fact, a path that does not start with a `/` should be specifically reported as invalid.
Also, file paths definitely don't end with a `/`.